stdlib/2and3/distutils/ccompiler.pyi

```diff
@@ -19,6 +19,16 @@ def new_compiler(plat: Optional[str] = ..., compiler: Optional[str] = ...,
 def show_compilers() -> None: ...
 
 class CCompiler:
+    dry_run: int
+    force: int
+    verbose: int
+    output_dir: str
```

Comment:
`Optional`

Reply:
hmmm I wasn't super certain about this one (seems to be late-init) but I'm not using the option so I changed it to your suggestion 👍